include/powsybl/iidm/converter/xml/CurrentLimitsXml.hxx

```diff
@@ -28,7 +28,7 @@ void CurrentLimitsXml::readCurrentLimits(CurrentLimitsAdder<S, O>& adder, const
     reader.readUntilEndElement(toString(CURRENT_LIMITS, index), [&adder, &reader]() {
         if (reader.getLocalName() == TEMPORARY_LIMIT) {
             const std::string& name = reader.getAttributeValue(NAME);
-            int acceptableDuration = reader.getOptionalAttributeValue(ACCEPTABLE_DURATION, std::numeric_limits<int>::max());
+            int acceptableDuration = reader.getOptionalAttributeValue(ACCEPTABLE_DURATION, std::numeric_limits<unsigned long>::max());
```

Comment:
If you change the numeric_limits type, you should probably also change the type of acceptableDuration.